oai_agents/gym_environments/base_overcooked_env.py

```diff
@@ -34,7 +34,7 @@
 class OvercookedGymEnv(Env):
     metadata = {'render.modes': ['human']}
 
-    def __init__(self, grid_shape=None, ret_completed_subtasks=False, stack_frames=False, is_eval_env=False,
+    def __init__(self, learner_type, grid_shape=None, ret_completed_subtasks=False, stack_frames=False, is_eval_env=False,
```

Comment:
if leaner_type does not have a default, we cannot run run_overcooked_game properly
suggested changes:
`def __init__(self, learner_type=LearnerType.ORIGINALER, grid_shape=None, ret_completed_subtasks=False, stack_frames=False, is_eval_env=False, shape_rewards=False, enc_fn=None, full_init=True, args=None, num_enc_channels=27, deterministic=False, **kwargs):`

Reply:
I fixed this by passing a custom learner_type in `overcooked_gui.py`. your suggested change would work but it's a risky fix because if we ever forget to pass the learner type when training the agents it would not run into an error and instead would run with the default value and we would not know that we have made a mistake.